### PR TITLE
Models and apis language corrections

### PR DESCRIPTION
**Pull request by mpgreaves**

### CODE REVIEW

**Comment by CsatariGergely** [comment]
Reviewdog result should be informative. I'm trying to find a way to _not_ fail it, but I'm not very successful yet.

File: content/en/docs/apis/_index.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ description: Reference for the Nephio models and APIs
...
 specified and maintained in the [Nephio API repository](https://github.com/nephio-project/api).
-This page aims to give a high level overview of the objects that are available and their
+This section aims to give a high-level overview of the objects that are available and their
 relationships, and is based on an
 [original document](https://docs.google.com/document/d/1-5nlpY4FbuhWtdKTvIqPOv4bWmA6zx6TdHoEfk9Jc_Q/edit)
 developed by Tal Liron.
 
-The aim is to keep these diagrams as simple as possible for now and only convey the important aspects of the modelled entities. As such they are intended to give a high-level overview of the entities and relationships that can be accessed and modified via the Nephio API, and provide reference to detailed documentation, generated from the code, where available.
+The aim is to keep the diagram below as simple as possible for now and only convey the important aspects of the modeled entities. As such, they are intended to give a high-level overview of the entities and relationships that can be accessed and modified via the Nephio API, and provide reference to detailed documentation, generated from the code, where available.
```
Comment (liamfallon) [inline content/en/docs/apis/_index.md:17]: modelled is the European english spelling but modeled is fine too ;-)

**Review by liamfallon — COMMENTED** [review]
/approve

**Pull request merged**